src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +98,27 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        return LocalDate.now().isAfter(someDate) ? someDate + " is before " + LocalDate.now()
```

Comment:
create a variable not to call LocalDate.now multiple times

Reply:
resolved